group_vars/aap/aap_config_controller_job_template.yml

```
@@ -0,0 +1,470 @@
+      spec:
+        - question_name: "Hostname or Group?"
+          question_description: "Please provide fully qualified hostname or group name"
+          required: true
+          type: text
+          variable: "host"
+          min: 0
+          max: 250
+
+  - name: "RHEL VM Update - Leapp Analysis"
+    description: "Job Template to leapp upgrade analysis"
+    playbook: playbooks/function_leapp_analysis.yml
+    labels:
+      - leapp
+    organization: "{{ aap_config_organization }}"
+    job_type: "{{ aap_config_default_job_run }}"
+    inventory: "{{ aap_config_default_inventory }}"
+    project: "{{ aap_config_default_project }}"
+    execution_environment: "{{ aap_config_default_execution_environment }}"
+    credentials:
+      - "Machine Credential"
+      - "Vault Credential"
+    notification_templates_success: "{{ aap_config_default_notification_template }}"
+    notification_templates_error: "{{ aap_config_default_notification_template }}"
+    allow_simultaneous: false
+    skip_tags: leapp_analysis
+
+  - name: "RHEL VM Update - Leapp Upgrade"
+    description: "Job Template to leapp upgrade"
+    playbook: playbooks/function_leapp_upgrade.yml
+    labels:
+      - leapp
+    organization: "{{ aap_config_organization }}"
+    job_type: "{{ aap_config_default_job_run }}"
+    inventory: "{{ aap_config_default_inventory }}"
+    project: "{{ aap_config_default_project }}"
+    execution_environment: "{{ aap_config_default_execution_environment }}"
+    credentials:
+      - "Machine Credential"
+      - "Vault Credential"
+    notification_templates_success: "{{ aap_config_default_notification_template }}"
+    notification_templates_error: "{{ aap_config_default_notification_template }}"
+    allow_simultaneous: false
+    skip_tags: leapp_upgrade
+
+  - name: "RHEL VM Update - Leapp 8.10 to 9.4 Remediate"
+    description: "Job Template to leapp remediate"
+    playbook: playbooks/function_leapp_remediate.yml
+    labels:
+      - leapp
+    organization: "{{ aap_config_organization }}"
+    job_type: "{{ aap_config_default_job_run }}"
+    inventory: "{{ aap_config_default_inventory }}"
+    project: "{{ aap_config_default_project }}"
+    execution_environment: "{{ aap_config_default_execution_environment }}"
+    credentials:
+      - "Machine Credential"
+      - "Vault Credential"
+    notification_templates_success: "{{ aap_config_default_notification_template }}"
+    notification_templates_error: "{{ aap_config_default_notification_template }}"
+    allow_simultaneous: false
+    skip_tags: leapp_remediate
+    survey_enabled: true
+    survey_spec:
+      name: "Survey for remediation"
+      description: "Define remediation points"
+      spec:
+        - question_name: "What do you want to remediate?"
+          question_description: "Inhibitors to remediate, refer to https://lachesis.internal.showroom.run/ui/repo/validated/infra/leapp/content/role/remediate/"
```

````
```suggestion
          question_description: "Inhibitors to remediate, refer to https://{{ groups.aap_pah | first }}/ui/repo/validated/infra/leapp/content/role/remediate/"
```